File: app/src/main/java/br/com/ifrs/meuifpoa/model/login/LoginRequest.java
package br.com.ifrs.meuifpoa.model.login;

/**
 * A classe `LoginRequest` encapsula os dados de uma requisição de login para o backend.
 * Contém os dados de autenticação, como nome de usuário e senha.
 */
public class LoginRequest {
    private String cpf;
    private String senha;

    /**
     * Construtor da classe `LoginRequest`.
     *
     * @param cpf   the cpf
     * @param senha the senha
     */
    public LoginRequest(String cpf, String senha) {
        this.cpf = cpf;
        this.senha = senha;
    }

    /**
     * Gets cpf.
     *
     * @return the cpf
     */
    public String getCpf() {
        return cpf;
    }

    /**
     * Gets senha.
     *
     * @return the senha
     */
    public String getSenha() {
        return senha;
    }
}

File: app/src/main/java/br/com/ifrs/meuifpoa/model/login/LoginRequest.kt
package br.com.ifrs.meuifpoa.model.login;

/**
 * Classe que representa uma requisição de login.
 */
public class LoginRequest {
    private String cpf;
    private String senha;

    /**
     * Construtor da classe LoginRequest.
     *
     * @param cpf o CPF do usuário
     * @param senha a senha do usuário
     */
    public LoginRequest(String cpf, String senha) {
        this.cpf = cpf;
        this.senha = senha;
    }

    /**
     * Retorna o CPF do usuário.
     *
     * @return o CPF do usuário
     */
    public String getCpf() {
        return cpf;
    }

    /**
     * Retorna a senha do usuário.
     *
     * @return a senha do usuário
     */
    public String getSenha() {
        return senha;
    }
}
